Examples/Framework/src/Framework/Sequencer.cpp

```diff
@@ -106,17 +132,108 @@ void Sequencer::addElement(std::shared_ptr<SequenceElement> element) {
   if (not element) {
     throw std::invalid_argument("Can not add empty/NULL element");
   }
+
+  bool valid = true;
+  std::string elementType{getAlgorithmType(*element)};
```

Comment:
```suggestion
  std::string elementType = getAlgorithmType(*element);
```
for consistency

Reply:
Doesn't actually work because `getAlgorithmType` returns `string_view`. Do you think I should change that too?

Reply:
oh okay no I think it is fine then